addons/common/XEH_postInit.sqf

```diff
@@ -254,12 +254,25 @@ GVAR(commonPostInited) = true;
 [{
     // If post inits are not ready then wait
     if !(SLX_XEH_MACHINE select 8) exitWith {};
+
     // If settings are not initialized then wait
-    if !(GVAR(SettingsInitialized)) exitWith {};
+    if (isNil QGVAR(settings)) exitWith {
+        diag_log text format["[ACE] Waiting on settings from server"];
```

Comment:
Do we want to print this to the rtp? What if the settings arrive late? Then we have spammed the rtp with quite a few messages. There is no need to print this multiple times I think.